doc/classes/AudioStreamPlayer2D.xml

```diff
@@ -139,30 +144,40 @@
 			<return type="void">
 			</return>
 			<description>
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="area_mask" type="int" setter="set_area_mask" getter="get_area_mask">
+			Areas in wich this sound plays.
 		</member>
 		<member name="attenuation" type="float" setter="set_attenuation" getter="get_attenuation">
+			Dampens Audio over distance with this as an exponent.
 		</member>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game.
 		</member>
 		<member name="bus" type="String" setter="set_bus" getter="get_bus">
+			Bus on which this audio is playing.
 		</member>
 		<member name="max_distance" type="float" setter="set_max_distance" getter="get_max_distance">
+			Sets the absolute maximum of the soundlevel, in dB.
 		</member>
 		<member name="playing" type="bool" setter="_set_playing" getter="is_playing">
+			If [code]'true'[/code], audio is playing.
 		</member>
 		<member name="stream" type="AudioStream" setter="set_stream" getter="get_stream">
+			Decides what is playing.
 		</member>
 		<member name="volume_db" type="float" setter="set_volume_db" getter="get_volume_db">
+			Base volume without dampening.
 		</member>
 	</members>
 	<signals>
 		<signal name="finished">
 			<description>
+				Fires when the audio stopped playing.
```

Comment:
"Emitted" is the preferred term, not "fires"

Also, you have mixed tense, should be "when the audio *stops* playing"